README.md

```diff
@@ -23,8 +23,10 @@ Join us on [Slack](https://slack.tmrow.com) if you wish to discuss development,
 The app let's you input trips manually by distance or time and transportation mode, and amount of passengers for a car trip.
 
 #### Automated
+- Automated Activity tracking
```

Comment:
this one is not open sourced. Do you still want to include it?

Reply:
I think it is valuable to let people know what is available in the app.